internal/services/ingest.go

```diff
@@ -38,7 +38,7 @@ var ErrAlreadyInSync = errors.New("ingestion is already in sync")
 const (
 	advisoryLockID                          = int(3747555612780983)
 	ingestHealthCheckMaxWaitTime            = 90 * time.Second
-	getLedgersLimit                         = 50 // NOTE: cannot be larger than 200
+	getLedgersLimit                         = 10 // NOTE: cannot be larger than 200. In production set it to 10 if your RPC has MAX_GET_LEDGERS_EXECUTION_DURATION="5s" or unset.
```

Comment:
I think you need to merge your changes from this PR to use the flag default value: https://github.com/stellar/wallet-backend/pull/296. Right?